app/Exceptions/Solutions/ManifestDoesntExistSolution.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Pterodactyl\Exceptions\Solutions;
+
+use Spatie\Ignition\Contracts\Solution;
+
+class ManifestDoesntExistSolution implements Solution
+{
+    public function getSolutionTitle(): string
+    {
+        return "The manifest.json file hasn't been generated yet";
+    }
+
+    public function getSolutionDescription(): string
+    {
+        return 'Run npm run build:production or yarn run build:production to build the frontend first.';
```

Comment:
Only suggest `yarn build:production`.  Nobody should be using `npm` as we only provide a `yarn.lock` file.  This may change later down the road if we switch to something like pnpm, but that will definitely be a v2 only kind of change.

Reply:
Good point!